### PR TITLE
Add AOCODAF405V3 target

### PR DESCRIPTION
**Pull request by ChenYang78256**
Add AOCODAF405V3

### CODE REVIEW

File: configs/AOCODAF405V3/config.h [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+#define MANUFACTURER_ID   SJET
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
```
Comment (lida2003) [inline configs/AOCODAF405V3/config.h:36]: Just keep consistancy with other F4V3 boards, add MS5611

```
#define USE_BARO_MS5611
```

File: configs/AOCODAF405V3/config.h [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    TIMER_PIN_MAP( 6, PA8 , 1,  1) \
+    TIMER_PIN_MAP( 7, PB10, 1,  0) \
+    TIMER_PIN_MAP( 8, PB11, 1,  0) \
+    TIMER_PIN_MAP( 9, PB1 , 2,  0)
+
+
+#define ADC_INSTANCE ADC1
+#define ADC1_DMA_OPT        0
+
+#define USE_BARO
```
Comment (lida2003) [inline configs/AOCODAF405V3/config.h]: Move `USE_BARO` before baro chip definitions.

File: configs/AOCODAF405V3/config.h [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+#define DEFAULT_CURRENT_METER_SCALE 500
+#define BEEPER_INVERTED
+#define SYSTEM_HSE_MHZ 8
+#define MAX7456_SPI_INSTANCE SPI2
+#define FLASH_SPI_INSTANCE SPI3
+#define USE_SPI_GYRO
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW90_DEG
```
Comment (lida2003) [inline configs/AOCODAF405V3/config.h:119]: Is there any mag chip supported? Please consult other F4V3 boards, and do self test, especially mag meter.

**Review by lida2003 — CHANGES_REQUESTED** [review]

**Comment by lida2003** [comment]
Please change PR tile from " Create config.h #2 " to "Add AOCODAF405V3 target"

File: configs/AOCODAF405V3/config.h (hunk 1 above)
Comment (ChenYang78256) [reply configs/AOCODAF405V3/config.h:36]: This MS5611 has been added.

File: configs/AOCODAF405V3/config.h (hunk 2 above)
Comment (ChenYang78256) [reply configs/AOCODAF405V3/config.h]: USE_BARO has been removed.

**Review by lida2003 — APPROVED** [review]

**Comment by ChenYang78256** [comment]
F405V3 Flash V2 self test passed

**Comment by lida2003** [comment]
Keep tracking this target board in https://github.com/Aocoda-RC/betaflight-config/pull/3

**Pull request closed without merging**